responses/nl/HassListAddItem.yaml

```diff
@@ -0,0 +1,5 @@
+language: "nl"
+responses:
+  intents:
+    HassListAddItem:
+      item_added: "{{ slots.item | capitalize }} toegevoegd"
```

Comment:
Is this actually correct to filter? It will mangle the input? (including abbreviations)

Reply:
Didn't think of abbreviations, I have removed the filter (also for the shopping list intent)